spec/infinispan_xsite_spec.js

```diff
@@ -22,6 +22,14 @@ describe('Infinispan xsite cluster client', function() {
     siteClients().then(function(cs) {
       expect(cs[0].getTopologyInfo().getMembers()).toEqual([t.earth1]);
       expect(cs[1].getTopologyInfo().getMembers()).toEqual([t.moon1]);
+      cs[0].getTopologyInfo().switchToCluster('site-moon-nonExistent')
```

Comment:
If the cluster does not exist, it should return `false`. You should check for that too.